website/docs/cli/commands/state/rm.mdx

```diff
@@ -9,7 +9,7 @@ description: >-
 The `terraform state rm` command removes the binding to an existing remote object without first destroying it. The remote object continues
 to exist but is no longer managed by Terraform.
 
-Instead of using the `terraform state rm` command, you can use `removed` blocks to remove resources. You can remove more than one resource at a time, and you can review removals as part of your normal plan and apply workflow. [Learn more about using `removed` blocks with resources](/terraform/language/resources/syntax#removing-resources) and [using `removed` blocks with modules](/terraform/language/modules/syntax#removing-modules).
+Instead of using the `terraform state rm` command, you can use `removed` blocks to remove resources. You can remove more than one resource at a time, and you can review removals as part of your normal plan and apply workflow. Refer to the [`removed` block reference](/terraform/language/block/removed) for additional information about removing resources. Refer to [Removing modules](/terraform/language/modules/syntax#removing-modules) for information about removing modules.
```

Comment:
We won't have a `removed` block reference yet though, right? So the `/terraform/language/block/removed` link won't work?

Reply:
Correct, but I think we'll have to revisit this link, anyway, because it would make more sense to link to the usage page on removing resources, which is still in review, instead of the `resource` reference.